### ISSUE TITLE
Postman 8.0 runner enhancement

### ISSUE THREAD

**Issue by ManiacDC** [a85/POSTMan-Chrome-Extension#9473]
**Is your feature request related to a problem? Please describe.**
Postman 8.0 moved the runner(s) to be in a normal tab. These get easily lost when many request tabs are also open.

**Describe the solution you'd like**
On the left sidebar, a new option for Runner, probably right above History or below Collections. This would also utilize a separate set of tabs.

**Describe alternatives you've considered**
Even the old method of a separate window was actually preferred IMO. I would often have the runner on a separate monitor so I could look at results and my requests at the same time. I realize I can open a new postman window to accomplish this.

**Comment by jonkrantz** [comment]
Yes - Please bring back the runner popping out in its own window.	This is a huge deal – Postman already had the issue of “too many tabs” and now we have added another tab.  The use of a separate window for the runner was ideal, it uses less monitor real estate than opening a second postman and allows working through failed collection much easier.

**Comment by sunnyttravers** [comment]
I agree. PLEEEEASE bring this feature back. I use collection runner a lot. It's much easier to have it in it's own separate window. Adding a drag and drop feature is not ideal. Having to scroll around to find the collection or folder is not ideal. I miss the old interface. It was much more manageable. Also, finding the collection runner is really hard. It is this tiny little link at the bottom of the window. I had to google for awhile to find it. You went from this wonderful big button at the top to a tiny little link that gets lost.

**Comment by likenspio** [comment]
About this, I'd like to suggest something like drag and drop the Runner tab to pop it out as a new window. Or some sort of configuration to choose if it opens in a new tab or new window. But preferable as a drag and drop feature. :D
